InternxtDesktop/AppDelegate.swift

```diff
@@ -53,10 +55,12 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     func applicationDidFinishLaunching(_ aNotification: Notification) {
         logger.info("App starting")
         ErrorUtils.start()
-    
+
         checkVolumeAndEjectIfNeeded()
+        backupsService.assignUrls()
```

Comment:
does this throws? If yes, move this to a method wrapped with a try catch, so the app start is not interrupted if this fails, if this doesn't throw, then it's ok

Reply:
this function does not throw any Error

Reply:
then it's ok